.github/actions/git-latest-tag/tests/git-latest-tag.bats

```diff
@@ -0,0 +1,35 @@
+setup() {
+    git fetch --tags
+
+    # Runs everywhere
+    DIR="$( cd "$( dirname "$BATS_TEST_FILENAME" )" >/dev/null 2>&1 && pwd )"
+    PATH="$DIR/..:$PATH"
+
+    # Mock GitHub Actions default variables
+    export GITHUB_ENV=/dev/null
+    export GITHUB_HEAD_REF="AAE-11760-get-latest-tag-sha"
+
+    # Mock git-latest-tag defaults
+    export PATTERN="*"
+    export REPO_DIR="$PWD"
+}
+
+@test "latest_tag" {
+    latest_tag=$(git tag --sort=-creatordate | head -n 1)
+    tag_sha=$(git rev-list -n 1 $latest_tag)
```

Comment:
it doesn't make sense to use in the test the same code used in the code under test to check the behavior.

you could verify that the message is structured correctly and do not check any specific tag/sha (that will change every time a new release is made)

Reply:
yeah i tought that was an unuseful test and actually I was wait for a suggestion 😂